src/test/java/seedu/address/logic/commands/schedule/DisplayScheduleBetweenCommandTest.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.logic.commands.schedule;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.commons.core.Messages.MESSAGE_EVENTS_LISTED_OVERVIEW;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalEvents.BIRTHDAY_PARTY;
+import static seedu.address.testutil.TypicalEvents.MUSICAL_COMPETITION;
+import static seedu.address.testutil.TypicalEvents.getTypicalEventBook;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.event.EventContainsKeyDateRangePredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code DisplayScheduleBetweenCommand}.
+ */
+public class DisplayScheduleBetweenCommandTest {
+
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+
+    private Model model = new ModelManager(getTypicalEventBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalEventBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        EventContainsKeyDateRangePredicate firstPredicate =
+                new EventContainsKeyDateRangePredicate(
+                        LocalDate.parse("10/12/2019", FORMATTER),
```

Comment:
Can possibly use the `ParserUtil#parseAnyDate` method that uses stricter constraints e.g must be a valid date, within the last 10 years etc